Controller/Callback/Offsite.php

```diff
@@ -143,6 +144,9 @@ public function execute()
                     case Truemoney::CODE:
                         $dispPaymentMethod = "True Money";
                         break;
+                    case Citipoints::CODE:
+                        $dispPaymentMethod = "Citi Points";
```

Comment:
Is this hard coded English?

Reply:
what is the question?

Reply:
those texts are added to the order log in the admin panel.

Reply:
Do they get translated?

Reply:
no, no necessary.